wee_alloc/Cargo.toml

```diff
@@ -23,11 +23,18 @@ size_classes = []
 # This is for internal use only.
 use_std_for_test_debugging = []
 
+static_array_backend = []
```

Comment:
This needs documentation in `wee_alloc/src/lib.rs`, and then you'll need to re-run `cargo readme`, which `build.sh` will do for you.

Let's also put this above the `use_std_for_test_debugging` feature, and give a quick one sentence summary of the feature right here, while we're at it.

Reply:
Can do!